bootloader/src/photon/bootloader_dct.c

```diff
@@ -0,0 +1,95 @@
+#include "bootloader_dct.h"
+
+#include "flash_mal.h"
+
+#include <stdint.h>
+#include <stddef.h>
+
+#define MODULAR_FIRMWARE 1
+#include "../../../hal/src/photon/ota_module_bounds.c"
+
+#define SYSTEM_PART2_MIN_MODULE_VERSION 107 // 0.7.0-rc.1
```

Comment:
Fixed in https://github.com/spark/firmware/pull/1325/commits/ca7537532d41d493abb7ae74548061c364f10f22